frontend/src/scenes/events/definitions/definitionDrawerLogic.ts

```diff
@@ -100,52 +173,93 @@ export const definitionDrawerLogic = kea<definitionDrawerLogicType<EventOrPropTy
         eventDefinitionTags: [
             () => [eventDefinitionsModel.selectors.eventDefinitions],
             (definitions: EventDefinition[]): string[] => {
-                const allTags = definitions
-                    .flatMap(({ tags }) => tags)
-                    .filter((a) => typeof a !== 'undefined') as string[]
+                const allTags = definitions.flatMap(({ tags }) => tags).filter((a) => !!a) as string[]
+                return uniqueBy(allTags, (item) => item).sort()
+            },
+        ],
+        eventPropertiesDefinitionTags: [
+            (selectors) => [selectors.eventPropertiesDefinitions],
+            (properties: PropertyDefinition[]): string[] => {
```

Comment:
You normally don't need to specify types in selectors. They are autodetected from the selector inputs... and adding something manually might lead to subtle bugs if something goes out of sync.